sys/include/net/gnrc/lorawan.h

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright (C) 2017 Fundación Inria Chile
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gnrc_lorawan GNRC LoRaWAN
+ * @ingroup     net_gnrc
+ * @brief       GNRC LoRaWAN stack implementation
+ *
+ * @{
+ *
+ * @file
+ * @brief   GNRC LoRaWAN API definition
+ *
+ * @author  José Ignacio Alamos <jose.alamos@haw-hamburg.de>
+ * @author  Francisco Molina <femolina@uc.cl>
+ */
+#ifndef NET_GNRC_LORAWAN_H
+#define NET_GNRC_LORAWAN_H
+
+#include "gnrc_lorawan_internal.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief maximum timer drift in percentage
+ *
+ * E.g a value of 0.1 means there's a positive drift of 0.1% (set timeout to
+ * 1000 ms => triggers after 1001 ms)
```

Comment:
add note that this is only a workaround in RIOT to compensate for bad timers/clocks, right?!

Reply:
https://github.com/RIOT-OS/RIOT/pull/11022/commits/1dec66fe2e7540c7d2e9609c49407a3f1216b404